src/extension/background-script/connectors/alby.ts

```diff
@@ -89,7 +89,7 @@ export default class Alby implements Connector {
 
   async getInvoices(): Promise<GetInvoicesResponse> {
     const incomingInvoices = (await this._request((client) =>
-      client.incomingInvoices({})
+      client.incomingInvoices({}, this._getRequestOptions())
```

Comment:
@pavanjoshi914 can you check if we can pass these options to the client so they don't need to be provided in every single request? best to keep it DRY